server/main.ts

```diff
@@ -21,26 +25,26 @@ async function bootstrap(): Promise<void> {
   server.enableCors();
 
   // improve security
-  server.use(require('helmet')());
+  server.use(helmet());
 
   // improve performance
-  server.use(require('compression')());
+  server.use(compression());
 
   // enable cookie
-  server.use(require('cookie-parser')());
+  server.use(cookieParser());
 
   // enable json response
   server.use(bodyParser.urlencoded({ extended: true }));
   server.use(bodyParser.json());
 
   // production ready session store
-  const pgSession = require('connect-pg-simple')(session);
+  const pgSession = connectPgSimple(session);
   server.use(
     session({
       secret: process.env.SESSION_SECRET as string,
       store: new pgSession({
         conString: `postgres://${process.env.DB_USERNAME}:${process.env.DB_PASSWORD}@${process.env.DB_HOST}:${process.env.DB_PORT}/${process.env.DB_DATABASE}`,
-        crear_interval: 60 * 60, // sec
+        // crear_interval: 60 * 60, // sec
```

Comment:
Could you tell me this optimization?

I set it according to cookie max age:

<https://github.com/saltyshiomix/ark/blob/f3ba9f7aad33188985c92c41e6a3a6d87f1a6a7a/server/main.ts#L54>